### ISSUE TITLE
About experiment results

### ISSUE THREAD

**Issue by sooonwoo** [a1600012888/clsa_pytorch#1]
Hi, thanks for your implementation!

Could you share the experimented results?

**Comment by a1600012888** [comment]
I can't reproduce the results in the paper due to some missing details about the hyper-param in the paper. 
current results: (linear evaluation protocol on ImageNet). (also on README.md)

|Train epochs   |  Single |  Mul-5 |  MoCo-v2 |   |
|---|---|---|---|---|
| 40  | 55.4% |  60.2% |  56.9% |   |
| 200  | 66.5%  |  68.3% | 67.6%  |   |
|   |   |   |   |   |

**Comment by sooonwoo** [comment]
Appreciate for sharing the result!

Have you ever experimented on other datasets such as CIFAR10/100?

**Comment by a1600012888** [comment]
Thanks. 
I didn't experimented on any other dataset.

**Comment by Isaver23** [comment]
> Appreciate for sharing the result!
> 
> Have you ever experimented on other datasets such as CIFAR10/100?

I had some experiments on cifar10 dataset. My baseline accuracy of MoCov2 on cifar10 dataset is 90.17(linear evaluation top1 accuracy, trained with freezed backbone), and my best clsa experiment result is 91.72.

**Comment by sooonwoo** [comment]
Hi, @Isaver23 

Appreciate for sharing!

**Comment by pikeyang** [comment]
> > Appreciate for sharing the result!
> > Have you ever experimented on other datasets such as CIFAR10/100?
> 
> I had some experiments on cifar10 dataset. My baseline accuracy of MoCov2 on cifar10 dataset is 90.17(linear evaluation top1 accuracy, trained with freezed backbone), and my best clsa experiment result is 91.72.

Could you share the code to train cifa10? thanks!
